run

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-exec gunicorn --log-level debug -w 4 -b 127.0.0.1:8000 teabot_endpoints.endpoints:app
+exec gunicorn --log-level debug -w 4 -b 0.0.0.0:${PORT:-8000} teabot_endpoints.endpoints:app
```

Comment:
This is all wild speculation but we have more than 4 endpoints which get updated simultaneously and only 4 workers, so we may have been relying on Nginx's queueing?